src/desktop_notifier/macos_support.py

```diff
@@ -54,15 +56,17 @@ def is_signed_bundle() -> bool:
     main_bundle = NSBundle.mainBundle
 
     if main_bundle.bundleIdentifier is None:
+        _log_unsigned_warning("bundleIdentifier is None")
```

Comment:
A warning is also emitted from `desktop_notifier.main` in this case, so this would be a bit redundant:

https://github.com/samschott/desktop-notifier/blob/2197773af7dec3a2910a1498ce1998f9405687b9/src/desktop_notifier/main.py#L86-L88